openml/tasks/functions.py

```diff
@@ -106,10 +106,18 @@ def _get_estimation_procedure_list():
 
     procs = []
     for proc_ in procs_dict["oml:estimationprocedures"]["oml:estimationprocedure"]:
+        task_type_int = int(proc_["oml:ttid"])
+        try:
+            task_type_id = TaskType(task_type_int)
+        except ValueError as e:
+            warnings.warn(
+                "Could not create task type id for %d due to error %s" % (task_type_int, str(e))
```

Comment:
I am not sure if we should raise a warning because there isn't a thing that users can do.
If you do want to raise a warning, I think it should be raised as a `RuntimeWarning`, instead of the default `UserWarning` (again, the user isn't at fault here). Related to this, I think the text message should then also be more explicit about this.

Reply:
Thanks for the feedback, just did so.